block-storage/src/main/scala/coop/rchain/blockstorage/FileLMDBIndexBlockStore.scala

```diff
@@ -139,6 +141,26 @@ class FileLMDBIndexBlockStore[F[_]: Monad: Sync: RaiseIOError: Log] private (
                })
     } yield result
 
+  def iterateStream: F[Stream[F, BlockMessage]] =
+    for {
+      indexes <- index.iterate { iterator =>
+                  Stream.chunk(
```

Comment:
Could you explain for education purpose - why this `Stream.chunk` and then `Chunk.iterable` wrappers used?

Reply:
https://fs2.io/#/guide?id=chunks

It is just a wrapper to convert `Iterable` to `Stream`